enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -586,7 +586,7 @@ func (c *FirecrackerContainer) LoadSnapshot(ctx context.Context, workspaceDirOve
 
 	var netNS string
 	if c.constants.EnableNetworking {
-		netNS = "/var/run/netns/" + c.id
+		netNS = "/var/run/netns/bb-executor-" + c.id
```

Comment:
wdyt about putting this in a func like `networking.NamespacePath(c.id)` so that the prefixing is transparent here?

Reply:
I thought you might ask for that :-P  Done